misk/src/main/kotlin/misk/config/ConfigModule.kt

```diff
@@ -24,6 +25,9 @@ class ConfigModule<T : Config>(
     bind<String>().annotatedWith<AppName>().toInstance(appName)
     bind(configClass).toInstance(config)
     bindConfigClassRecursively(configClass)
+
+    multibind<WebActionEntry>().toInstance(
```

Comment:
Is this necessary given that it is also bound in `ConfigWebModule`? Think we should leave the low level `ConfigModule` independent of web concerns.